com.yakindu.sct.examples.coffeemachine.cpp/src/sc_timer_service.cpp

```diff
@@ -39,6 +39,9 @@ void TimerService::unsetTimer(TimedInterface *statemachine, sc_eventid event) {
 	for (it = this->timer_tasks.begin(); it != timer_tasks.end(); ++it){
 		if(it->pt_evid == event){
 			pthread_cancel(it->pt);
+			if(pthread_cancel(it->pt) != ESRCH) {
```

Comment:
Don't you try to cancel twice here? Line 41 can be removed then, no?

Reply:
ups, yep it must be removed!